omedev/docker-prod-apps.yml

```diff
@@ -262,6 +263,16 @@
           - grafana-data:/var/lib/grafana
       register: _grafana_container
 
+    - name: Run docker omero grafana json datasource
+      become: true
+      docker_container:
+        image: registry.gitlab.com/openmicroscopy/incubator/omero-grafana-json-datasource:0.1.1
```

Comment:
Does this need migrating?

Reply:
Depends on whether you want to support and promote it 😄